pkg/kv/kvserver/raftlog/iter_bench_test.go

```diff
@@ -171,8 +171,7 @@ func BenchmarkIterator(b *testing.B) {
 			if ok, err := it.Next(); err != nil || !ok {
 				return false, err
 			}
-			ent := it.Entry()
-			ent.Release()
+			_ = it.Entry()
```

Comment:
This isn't really `NextPooled` any more since there isn't any pooling? Might as well remove this then.

Reply:
Done.